src/json-rpc/process-response.ts

```diff
@@ -109,8 +109,7 @@ function isJsonRpcErrorResponse(
       typeof error.data.type === "string" &&
       error.data.code &&
       typeof error.data.code === "string" &&
-      (error.data.url === undefined ||
-        (error.data.url && typeof error.data.url === "string"))
+      (error.data.url === undefined || (typeof error.data.url === "string"))
```

Comment:
The previous value would only pass for non-empty strings. If this was intended, please let me know and we can sort out the best approach for these handling undefined/null/empty string values.

Reply:
The API should either return `null` or a valid URL, not an empty string.  This is defined in the OpenRPC definition as well, so an empty string would fail response validation

Reply:
Perfect. I'll update the validation in the SDK to accept a null value and remove the default empty string I added on our side.